library/src/main/scala/scala/scalajs/js/typedarray/DataViewExt.scala

```diff
@@ -33,10 +33,12 @@ object DataViewExt {
     @inline
     def setInt64(index: Int, value: Long,
         littleEndian: Boolean = false): Unit = {
+      // scalafmt: { maxColumn = 85 }
       val high = (value >>> 32).toInt
       val low = value.toInt
       dataView.setInt32(index + (if (littleEndian) 4 else 0), high, littleEndian)
-      dataView.setInt32(index + (if (littleEndian) 0 else 4), low,  littleEndian)
+      dataView.setInt32(index + (if (littleEndian) 0 else 4), low, littleEndian)
+      // scalafmt: { maxColumn = 80 }
```

Comment:
Yes, the formatting change is in the wrong commit.

The line length limit change is a poor man's implementation of the *soft* limit at 80 characters. The wrapping to stay under 80 characters is significantly worse than going a few characters over the limit. The soft limit problem is filed as https://github.com/olafurpg/scalafmt/issues/232, although it has been closed as Won't fix.